modules/040-node-manager/images/bashible-apiserver/pkg/template/context_builder.go

```diff
@@ -438,6 +442,32 @@ type bashibleContext struct {
 	Proxy    map[string]interface{}       `json:"proxy" yaml:"proxy"`
 }
 
+func (bc *bashibleContext) AddToChecksum(checksumCollector hash.Hash) error {
+	cpy := deepcopy.Copy(bc)
+	bcCopy := cpy.(*bashibleContext)
+
+	cloudInstancesRaw, ok := bcCopy.NodeGroup["cloudInstances"]
+	if ok {
+		cloudInstances, ok := cloudInstancesRaw.(map[string]interface{})
+		if ok {
+			// remove counter for prevent updating nodes while upscale
+			delete(cloudInstances, "maxPerZone")
+			delete(cloudInstances, "maxSurgePerZone")
+			delete(cloudInstances, "maxUnavailablePerZone")
+			delete(cloudInstances, "minPerZone")
```

Comment:
I guess we can also delete `zones` key from this map.
When we change the zones array - why do nodes should rollout?